maven/pom.xml

```diff
@@ -184,6 +184,7 @@
 							<redirectTestOutputToFile>true</redirectTestOutputToFile>
 							<showEclipseLog>false</showEclipseLog>
 							<testFailureIgnore>false</testFailureIgnore>
+							<failIfNoTests>false</failIfNoTests>

```

Comment:
Thats a cool flag. If we have this, do we still need the dummy template test?

Reply:
Unfortunately yes, because it just avoids a fail but still throws a warning, which makes the maven_build script fail. Of course we could add the warning to the exceptions, but from a maven point of view, with convention over configuration I prefer the template test for the moment.

Reply:
Alright!